html/elements/iframe.json

```
@@ -1027,6 +1027,55 @@
             }
           }
         },
+        "sandbox-allow-same-origin": {
+          "__compat": {
+            "description": "<code>sandbox=\"allow-same-origin\"</code>",
+            "support": {
+              "chrome": {
+                "version_added": true
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": true
+              },
+              "firefox_android": {
+                "version_added": true
+              },
+              "ie": {
+                "version_added": true
+              },
+              "opera": {
+                "version_added": true
+              },
+              "opera_android": {
+                "version_added": true
+              },
+              "safari": {
+                "version_added": true,
+                "notes": "Safari will block script execution without 'allow-scripts' even if 'allow-same-origin' is set. For example, any bound handlers for click events of nodes inside an iframe will throw an Error of blocking script execution."
```

suggestion: writing style and markup

Nothing major here. For consistency with other notes, we'll use the present tense and mark up keywords with code.

```suggestion
                "notes": "Safari blocks script execution without <code>allow-scripts</code> even if <code>allow-same-origin</code> is set. For example, any bound handlers for click events of nodes inside an iframe throw an error for blocked script execution."
```

---

Well done!